src/main/java/com/google/api/generator/engine/ast/CommentStatement.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import javax.annotation.Nullable;
+
+@AutoValue
+public abstract class CommentStatement implements Statement {
+
+  public abstract ImmutableList<LineComment> lineComments();
```

Comment:
As discussed today and previous PRs, `CommentStatement` is a thin wrapper for `Comment`. See my comment in PR #30: "In the future, there should be a CommentStatement that lightly wraps Comment."

Reply:
I got the point and updated the API/unit tests thank you!